experimental/casemap/Cargo.toml

```diff
@@ -37,6 +37,7 @@ icu_provider = { version = "1.2.0", path = "../../provider/core", features = ["m
 yoke = { version = "0.7.1", path = "../../utils/yoke", features = ["derive"] }
 zerovec = { version = "0.9.4", path = "../../utils/zerovec", features = ["yoke"] }
 writeable = { version = "0.5.1", path = "../../utils/writeable" }
+icu_properties = { path = "../../components/properties" }
```

Comment:
Suggestion (optional): maybe it should be an optional feature since clearly CaseMapper is useful without this dep

Reply:
Yeah, we could, but overall I think depending on properties is not a huge deal and I consider that crate to be a pseudo-core dependency anyway.